met-web/src/components/engagement/view/EngagementView.tsx

```diff
@@ -66,6 +67,27 @@ export const EngagementView = () => {
                     </Grid>
                     <Grid item xs={12} lg={4}>
                         <WhoIsListeningWidget />
+                        <Button
+                            onClick={() =>
+                                dispatch(
```

Comment:
Let's move this to a handle function above so it doesn't bloat the JSX. Also let's not use a hierarchy, data: {header, ..}. jsut header, description, handleClose, handleConfirm, type